pkg/controller/master/upgradelog/common.go

```diff
@@ -238,6 +238,22 @@ func prepareClusterFlow(upgradeLog *harvesterv1.UpgradeLog) *loggingv1.ClusterFl
 						ContainerNames: []string{"upgrade"},
 					},
 				},
+				{
+					ClusterSelect: &loggingv1.ClusterSelect{
+						Labels: map[string]string{
+							"app": "fleet-controller",
+						},
+						Namespaces: []string{"cattle-fleet-agent"},
```

Comment:
Are both namespaces right?

`cattle-fleet-system/fleet-controller and cattle-fleet-local-system/fleet-agent`

Reply:
Updated, thanks!